bombastic/api/src/server.rs

```diff
@@ -69,6 +82,19 @@ async fn health() -> HttpResponse {
     HttpResponse::Ok().finish()
 }
 
+#[utoipa::path(
+    get,
+    path = "/api/v1/sbom",
+    responses(
+        (status = 200, description = "SBOM found"),
+        (status = NOT_FOUND, description = "SBOM not found in archive"),
+        (status = BAD_REQUEST, description = "Missing valid purl or index entry"),
+    ),
+    params(
+        ("purl" = String, Query, description = "Package URL of SBOM to query"),
+        ("sha256" = String, Query, description = "The index value, as a sha256"),
```

Comment:
We should remove all references to sha256, this will be supported via search later on and is not implemented today.